inginious/frontend/plugins/UNCode/static/js/uncode.js

```diff
@@ -8,16 +8,6 @@ jQuery(document).ready(function () {
         logoElement.html(image).append(" Code");
     }
 
-    function updateNavbar(){
-        // Update navbar with new information.
-         const navbar = $("#wrapper")
-            .find("> div.navbar.navbar-default.navbar-static-top");
-         //Add UNCode button
-         navbar.find("> div > div.collapse.navbar-collapse > ol.nav.navbar-nav.navbar-right")
-             .append("<li><a class='navbar-link' href='https://juezun.github.io/' target='_blank'>" +
-                 "<i class='fa fa-question-circle-o' style='margin-right: 5px' aria-hidden='true'></i>About UNCode</a></li>");
-    }
-
```

Comment:
Fix this, I think this is new in PR#283 and probably you did not correctly merge master.

Reply:
Yes, i tried to merge master into translations branch, however some files couldn't be merged automatically, so i had to merge manually.